index.html

```diff
@@ -0,0 +1,152 @@
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Tangerine">
+    <title>MEdicore</title>
+    <link href="./styles.css" rel="stylesheet">
+</head>
+<body>
+    <div class="section--pink">
+        <div class="menu-bar">
+            <div class="menu-bar__section menu-bar__section--text-big">
+                <div class="menu-bar__link">Medical Core</div>
+            </div>
+            <div class="menu-bar__section menu-bar__section--text-muted menu-bar__links">
+                <div class="menu-bar__link">Home</div>
+                <div class="menu-bar__link">Product</div>
+                <div class="menu-bar__link">Pricing</div>
+                <div class="menu-bar__link">Contact</div>
+            </div>
+            <div class="menu-bar__section">
+                <button class="button button--white">Login</button>
+                <button class="button button--uppercase button--blue">join us <img src="./images/Vector.png"> </button>
+            </div>
+        </div>
+        <div class="welcome">
+            <div class="welcome__content">
+                <label class="welcome__text--blue welcome__text--big-in-sm">Welcome</label>
+                <label class="welcome__text--big">A Great Place to Receive Care</label>
+                <label class="welcome__text--muted">Get your best looking smile now!</label>
+                <div class="welcome__buttons">
+                    <button class="button button--blue">Join Us</button>
+                    <button class="button button--border">Learn More</button>
+                </div>
+            </div>
+            <div class="welcome__empty"></div>
+        </div>
+    </div>
+    <div class="content">
+        <div class="content__image">
+            
+            <img class="content__image--1" src="./images/lady-doctor.png">
+        </div>
+        <div class="content__text">
+            <div>
+                <hr class="content__hr">
+                <label class="content__text--big">Meet our experts</label>
+                <div class="content__text--muted">
+                    Problems trying to resolve the conflict between 
+                    the two major realms of Classical physics:
+                    Newtonian mechanics 
+                    <div class="welcome__text--blue text--padded">
+                        Learn More <img src="./images/Vector.png">
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="teams">
+        <div class="teams__text">
+            <label class="text--padded welcome__text--blue">
+                Team
+            </label>
+            <label class="content__text--big">Our Team</label>
+            <div class="content__text--muted">
+                Problems trying to resolve the conflict between 
+                the two major realms of Classical physics: Newtonian mechanics 
+            </div>
```

Comment:
Shouldn't use modifiers without specifying the block or the element class. Good practice would be something like `<label class="content__text content__text--big"`